Utils/constants.js

```diff
@@ -3,6 +3,15 @@ const TalentBoardUserStatus = {
   EXPIRED: 2
 };
 
+const visibilityDurationMapping = {
```

Comment:
```suggestion
const VisibilityDurationMapping = {
```
To keep it consistent with the other constant variable names

Reply:
done